doc/endpoint/codecs.md

```diff
@@ -88,6 +88,89 @@ Different codec formats can be used in different contexts. When defining a path,
 with the `TextPlain` media type can be used. However, for bodies, any media type is allowed. For example, the 
 input/output described by `jsonBody[T]` requires a json codec.
 
+## Handling Delimited Path Parameters
```

Comment:
Thanks for the proposal! I think I'd suggest moving it to a top-level page linked in the `How To` section. The "codec" section over here is more of a reference, and this describes a specific use-case

Reply:
I see the `How To` section references links to `.scala` files. To follow that same pattern I'm thinking I'll convert this to `DelimitedPathParameterExample.scala` and add code comments. Unless I'm missing something and I can just attach a `.md`

Reply:
Oh no, I meant a regular .md page. There are links to examples in the `Examples` section, but you should be able to  add a link here: https://github.com/softwaremill/tapir/blob/master/doc/index.md?plain=1#L158 to the new page

Reply:
I wasn't too sure where it should be housed so I made a dir for how-tos and placed it in there. Let me know if I should change that or rename it.